submission/submission-security/src/test/kotlin/ebi/ac/uk/security/service/SecurityServiceTest.kt

```diff
@@ -259,4 +263,28 @@ internal class SecurityServiceTest(
             }
         }
     }
+
+    @Nested
+    inner class CheckToken {
+
+        private val securityToken = mockk<SecurityToken>()
+        private val user = mockk<User>()
+
+        @Test
+        fun `check when is not in black list`() {
```

Comment:
We should add a test to verify every time a token is created it has a creation time which is the variation